<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="confirmButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>400</y>
      <width>300</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm IP</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="IP_3_textArea">
    <property name="geometry">
     <rect>
      <x>415</x>
      <y>300</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTextEdit" name="IP_4_textArea">
    <property name="geometry">
     <rect>
      <x>525</x>
      <y>300</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="dotLabel1">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>300</y>
      <width>10</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QLabel" name="dotLabel3">
    <property name="geometry">
     <rect>
      <x>505</x>
      <y>300</y>
      <width>10</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QLabel" name="dotLabel2">
    <property name="geometry">
     <rect>
      <x>395</x>
      <y>300</y>
      <width>10</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QLabel" name="instructionLabel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>240</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>INSERT IP ADRESS</string>
    </property>
   </widget>
   <widget class="QLabel" name="instructionLabel_2">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>300</y>
      <width>51</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>192</string>
    </property>
   </widget>
   <widget class="QLabel" name="instructionLabel_3">
    <property name="geometry">
     <rect>
      <x>325</x>
      <y>300</y>
      <width>51</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>168</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cameraButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>150</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Turn On/Off Camera</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
